app/presenters/vendor_api/hesa_qualification_fields_presenter.rb

```diff
@@ -8,11 +8,11 @@ def to_hash
       return {} unless @qualification.level == 'degree'
 
       {
-        hesa_degtype: @qualification.qualification_type_hesa_code,
-        hesa_degsbj: @qualification.subject_hesa_code,
-        hesa_degclss: @qualification.grade_hesa_code,
-        hesa_degest: @qualification.institution_hesa_code,
-        hesa_degctry: @qualification.institution_country,
+        hesa_degtype: @qualification.qualification_type_hesa_code&.to_s.rjust(3, '0'),
+        hesa_degsbj: @qualification.subject_hesa_code&.to_s.rjust(6, '0'),
+        hesa_degclss: @qualification.grade_hesa_code&.to_s.rjust(2, '0'),
+        hesa_degest: @qualification.institution_hesa_code&.to_s.rjust(4, '0'),
+        hesa_degctry: @qualification.institution_country&.to_s.rjust(2, '0'),
```

Comment:
These are 2-char ISO codes so shouldn't need padding

Reply:
The refactored version of this code doesn't pad `institution_country`.